### ISSUE TITLE
BUG: Can't set ETS_TOOLKIT variable

### ISSUE THREAD

**Issue by Eric89GXL** [spyder-ide/spyder#4399]
## Description of your problem

**What steps will reproduce the problem?**

1. Install Spyder + mayavi + wxpython with Anaconda.
2. Open Spyder->Preferences->Python console
3. See that the ETS_TOOLKIT setting is grayed out

**What is the expected output? What do you see instead?**

1. I should be able to set ETS_TOOLKIT to e.g. `wx`. Running scripts with `ETS_TOOLKIT=wx python -i ...` works fine, Spyder should be able to run scripts this way, too.

2. If I can't set ETS_TOOLKIT, it would be nice to know why via a tooltip (though not as critical if #1 is fixed).

## Versions and main components

* Spyder Version: 3.1.3
* Python Version: 2.7.13
* Qt Version:
* PyQt Version:
* Operating system: Linux x86_64

If this behavior is unexpected, I'm happy to try fixing it.

## Dependencies
```
jedi =0.9.0      :  0.9.0 (OK)
matplotlib >=1.0 :  1.5.1 (OK)
nbconvert >=4.0  :  4.2.0 (OK)
numpy >=1.7      :  1.11.3 (OK)
pandas >=0.13.1  :  0.19.2 (OK)
pep8 >=0.6       :  1.7.0 (OK)
psutil >=0.3     :  5.0.1 (OK)
pyflakes >=0.5.0 :  1.5.0 (OK)
pygments >=2.0   :  2.1.3 (OK)
pylint >=0.25    :  1.6.4 (OK)
qtconsole >=4.2.0:  4.2.1 (OK)
rope >=0.9.4     :  0.9.4 (OK)
sphinx >=0.6.6   :  1.5.1 (OK)
sympy >=0.7.3    :  1.0 (OK)
```

**Comment by ccordoba12** [comment]
> If this behavior is unexpected, I'm happy to try fixing it.

Yes, it is. But the Python console is going to be removed in Spyder 3.2, so you would need to add this to the IPython console instead.

Is that Ok for you?

**Comment by Eric89GXL** [comment]
Ahh that's a bummer, I always use vanilla Python terminals to ensure full compatibility. But I guess it shouldn't matter much.

I'll try to look into it soon. I use `mayavi` + `matplotlib` a lot and they don't play nicely when using the same (default Qt) backend.

**Comment by Eric89GXL** [comment]
I'll tackle this once #4418 is fixed
